### PR TITLE
changes defibs to be less strong

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
halfs both timers for defibs
removes fixing time-of-death via surgery

### CODE REVIEW

File: code/modules/surgery/wound_repair.dm [hunk 1]
```diff
@@ -202,7 +202,7 @@
 		SPAN_NOTICE("You finish filtering out any toxins to [target]'s body and repairing any neural degradation with the [tool_name].") )
```
Comment (WatermelonsEverywhere) [inline code/modules/surgery/wound_repair.dm:202]: All neural degradation repair mentions should be removed if it no longer happens to avoid confusion.

Comment (Trilbyspaceclone) [reply code/modules/surgery/wound_repair.dm:202]: Correct, good call

File: code/modules/surgery/wound_repair.dm [hunk 2]
```diff
@@ -199,10 +199,10 @@
 		if(advanced_medical)
 			heal_amount -= calculate_expert_surgery_bonus(user) * 2
 		user.visible_message(SPAN_NOTICE("[user] finishes [advanced_medical ? "expertly" : ""] filtering out any toxins in [target]'s body and repairing any neural degradation with the [tool_name]."), \
```
Comment (WatermelonsEverywhere) [inline code/modules/surgery/wound_repair.dm:201]: There's three other degeneration mentions on lines 181, 182 and 201 which should be all of them.

**Comment by Hlscientist** [comment]
Works

**Pull request merged**